src/rtree.ml

```diff
@@ -178,6 +178,15 @@ module Make (E : Envelope) (V : Value with type envelope = E.t) = struct
       0 partitions lst
     |> List.map List.rev
     |> List.filter_map (fun v -> if v = [] then None else Some v)
+  
+  (* Rtree Iter Function *)
+  let rec _iter rtree f =
```

Comment:
```suggestion
  let rec iter rtree f =
```

The function should be called `iter`. The reason I think you were getting an unused variable warning is because the function was not exposed in the interface. You will need to add the iter function's type to `Rtree_intf` so you can use it outside of the library. You can read more about that here: https://dev.realworldocaml.org/files-modules-and-programs.html

Reply:
Oh okay thank you very much, I'll definitely do that.